client/.gitignore

```diff
@@ -4,6 +4,7 @@
 /node_modules
 /.pnp
 .pnp.js
+.eslintrc.json
```

Comment:
I think we should all be sharing the eslint config, if we remove it we will end up having different settings each!

Reply:
Ahh remove it then! Sorry, just popped it in there in case the missing commas played up again!